ParisBNF/abb/BNFabb88.xml

```diff
@@ -32,20 +32,41 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                     <msContents>
                         <summary/>
                         <msItem xml:id="ms_i1">
-                            <locus from="5r" to="172ra"/>
-                            <title type="incomplete" ref="LIT1509GadlaW">Vita and miracles of Walatta Ṗeṭros</title>
-                            <textLang mainLang="gez"/>
+                            <locus from="5r" to="178r"/>
+                                <title ref="LIT7229DossierWP"/>
+                            <msItem xml:id="ms_i1.1">
```

Comment:
add a title for i1.1 (e.g. Life and Miracles of WP), it cannot be without a title